packages/dx-react-chart-demos/src/demo-sources/bar-chart/series-interaction.jsxt

```diff
@@ -0,0 +1,202 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Chart,
+  BarSeries,
+  ArgumentAxis,
+  ValueAxis,
+  Title,
+  Legend,
+  Tooltip,
+} from '@devexpress/dx-react-chart-<%&themeName%>';
+import * as d3Format from 'd3-format';
+import { scaleBand } from '@devexpress/dx-chart-core';
+import {
+  ArgumentScale, Stack, Animation, EventTracker, HoverState, SelectionState,
+} from '@devexpress/dx-react-chart';
+
+import { annualVehiclesSales as data } from '../../../demo-data/data-vizualization';
+
+const tooltipContentTitleStyle = {
+  fontWeight: 'bold',
+  paddingBottom: 0,
+};
+const tooltipContentBodyStyle = {
+  paddingTop: 0,
+};
+const formatTooltip = d3Format.format(',.2r');
+const TooltipContent = (props) => {
+  const { targetItem, text, ...restProps } = props;
+  return (
+    <div>
+      <div>
+        <Tooltip.Content
```

Comment:
Tooltip.Content is undocumented. Maybe we should do that?

Reply:
It is already partially [documented](https://devexpress.github.io/devextreme-reactive/react/chart/docs/reference/tooltip/#tooltip.contentprops). Let's fix it in a separate PR.

Reply:
ok